apis/core/lib/domain/dataset/update.ts

```diff
@@ -55,5 +58,10 @@ export async function update({
     })
   })
 
+  const project = await store.get(await findProject({ dataset }), { allowMissing: true })
+  project
+    ?.deleteOut(lindasSchema.datasetNextDateModified)
+    .addOut(lindasSchema.datasetNextDateModified, datasetResource.out(lindasSchema.datasetNextDateModified))
+
```

Comment:
Here is important that when a cube metadata is saved, the "next planned date" is copied to the project itself so that it will be possible to sort and display it in the project list

I would prefer event-driven or  a more flexible way to retrieve project collection but not an option in current codebase

Reply:
For sure, that would be a big change in the codebase.

For more complex workflows, one could use an out-of-the-box job scheduler (with the added dependencies)